### PR TITLE
Standardized docker scheme

### PR DESCRIPTION
**Pull request by ericonr**
This uses https://github.com/cnpem/epics-in-docker/pull/4 , so we should merge that first.

For comparison:

- current image: 515MB
- new image: 140MB

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,6 @@ E.g.:
 
 ### How to build docker image
 
-Run `docker compose` from docker directory saving COMMIT_ID as environemnt variable:
+Run `docker compose` from root directory, after exportin the `COMMIT_ID` environemnt variable:
```
Comment (mmdonatti) [inline README.md:24]: two typos here: exporting and environment

File: README.md
Comment (mmdonatti) [inline README.md:1]: I suggest to remind to init and update submodules as well, or clone recursively

**Comment by mmdonatti** [comment]
build issue:
- Docker version 23.0.2, build 569dd73
- Debian 11.6

```
mauricio.donatti@s-gie01-l:~/projects/sinap/sinap-timing-epics-ioc$ COMMIT_ID=$(git rev-parse --short HEAD) docker compose build
[+] Building 3.5s (11/15)
 => [internal] load .dockerignore                                                                                                                                       0.3s
 => => transferring context: 2B                                                                                                                                         0.0s
 => [internal] load build definition from Dockerfile                                                                                                                    0.4s
 => => transferring dockerfile: 788B                                                                                                                                    0.0s
 => [internal] load metadata for ghcr.io/cnpem/lnls-debian-11-epics-7:latest                                                                                            0.4s
 => [internal] load metadata for docker.io/library/debian:11.7-slim                                                                                                     1.0s
 => [stage-1 1/5] FROM docker.io/library/debian:11.7-slim@sha256:924df86f8aad741a0134b2de7d8e70c5c6863f839caadef62609c1be1340daf5                                       0.0s
 => [internal] load build context                                                                                                                                       0.2s
 => => transferring context: 15.36kB                                                                                                                                    0.0s
 => [build_stage 1/5] FROM ghcr.io/cnpem/lnls-debian-11-epics-7:latest@sha256:ef73167bde992b7990a0a2baadf709e02e306ae1e7be7946da038003398189fd                          0.0s
 => CACHED [stage-1 2/5] RUN apt update -y && apt install -y --no-install-recommends busybox netcat-openbsd procserv && apt clean && rm -rf /var/lib/apt/lists/*        0.0s
 => CACHED [build_stage 2/5] WORKDIR /opt/sinap-timing-epics-ioc                                                                                                        0.0s
 => CACHED [build_stage 3/5] COPY . .                                                                                                                                   0.0s
 => ERROR [build_stage 4/5] RUN echo STATIC_BUILD=YES >> configure/CONFIG_SITE.local && cp /opt/epics/RELEASE configure/RELEASE                                         1.8s
------
 > [build_stage 4/5] RUN echo STATIC_BUILD=YES >> configure/CONFIG_SITE.local && cp /opt/epics/RELEASE configure/RELEASE:
#0 0.879 cp: cannot stat '/opt/epics/RELEASE': No such file or directory
------
failed to solve: process "/bin/sh -c echo STATIC_BUILD=YES >> configure/CONFIG_SITE.local && cp /opt/epics/RELEASE configure/RELEASE" did not complete successfully: exit code: 1
```

File: iocBoot/ioctiming/runProcServ.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ set -u
 
 # Run run*.sh scripts with procServ
 if [ "${UNIX_SOCKET}" ]; then
-    /usr/bin/procServ -f -n TIMING_${DEVICE_TYPE} -i ^C^D unix:./procserv.sock ./runGenericModule.sh "$@"
+    /usr/bin/procServ -L-  -f -n TIMING_${DEVICE_TYPE} -i ^C^D unix:./procserv.sock ./runGenericModule.sh "$@"
```
Comment (mmdonatti) [inline iocBoot/ioctiming/runProcServ.sh:23]: this shouldn't be `/usr/bin/procServ -L -f -n TIMING_${DEVICE_TYPE} -i ^C^D unix:./procserv.sock ./runGenericModule.sh "$@"` ?

**Review by mmdonatti — COMMENTED** [review]
We need to install python3 to run [event_alias.py](https://github.com/lnls-dig/sinap-timing-epics-ioc/blob/master/iocBoot/ioctiming/event_alias.py). 

This script runs only for EVG to automatically create event_alias.cmd file, linking high level event names to event numbers based on control system constants [high level events table](https://github.com/lnls-sirius/control-system-constants/blob/master/timesys/high-level-events.py).

Another option is to create a bash script to do this task.

File: README.md
Comment (ericonr) [reply README.md:1]: What do you think about my addition?

File: iocBoot/ioctiming/runProcServ.sh (hunk 2 above)
Comment (ericonr) [reply iocBoot/ioctiming/runProcServ.sh:23]: No.

`-L` is the flag to enable logging, and as an argument it takes the file to log to, or `-` to specify logging to `stdout`.

**Comment by ericonr** [comment]
This is a TODO task for [epics-in-docker](https://github.com/cnpem/epics-in-docker): installation of arbitrary build time and runtime dependencies

I will try and add this functionality soon!

File: README.md
Comment (mmdonatti) [reply README.md:1]: perfect!

**Review by mmdonatti — APPROVED** [review]
I think we're ready to merge

**Pull request merged**